src/panels/config/entities/entity-registry-settings-editor.ts

```diff
@@ -257,6 +264,16 @@ export class EntityRegistrySettingsEditor extends LitElement {
     });
   }
 
+  private _testDefaultCode(
+    codeFormat?: string,
+    value?: string | null
+  ): boolean {
+    if (codeFormat && value) {
+      return RegExp(codeFormat).test(value) !== true;
```

Comment:
```suggestion
      return RegExp(codeFormat).test(value);
```

Reply:
Need to flip the logic here or it will show the error message until the correct length is applied.
Any other suggestion that above?

Reply:
Oh sorry 🙃 :
`return !RegExp(codeFormat).test(value);`

Reply:
Or name it `_isInvalidDefaultCode`

Reply:
Thanks!